client/state/selectors/get-user-settings.ts

```diff
@@ -0,0 +1,17 @@
+import type { UserSettingValue } from 'calypso/state/selectors/get-user-setting';
+import type { AppState } from 'calypso/types';
+
+import 'calypso/state/user-settings/init';
+
+export type UserSettingsType = Record< string, UserSettingValue >;
+
+/**
+ * Returns all user settings as one object
+ *
+ *
+ * @param {AppState} state Global state tree
+ * @returns {UserSettingsType | null} dictionary with the setting names and values
+ */
```

Comment:
Same thing, I would say that we can remove it and still self descriptive enough, isn't it?